function-reference/store-value.md

```diff
@@ -0,0 +1,22 @@
+---
+description: >-
+  This function is used to store key value data within browser local storage. `appsmith.store.key` can be used to fetch the stored data.
+---
+
+# Store Value 
+
+## Signature
+
+```text
+storeValue(key: string, value: any) -> void"
```

Comment:
Can you please remove double quote at the end? Doesn't seem relevant.

Reply:
Done